localstack/services/cloudwatch/provider_v2.py

```diff
@@ -457,6 +476,36 @@ def describe_alarms(
         composite_alarms = [a for a in alarms if a.get("AlarmRule") is not None]
         return DescribeAlarmsOutput(CompositeAlarms=composite_alarms, MetricAlarms=metric_alarms)
 
+    def describe_alarms_for_metric(
+        self,
+        context: RequestContext,
+        metric_name: MetricName,
+        namespace: Namespace,
+        statistic: Statistic = None,
+        extended_statistic: ExtendedStatistic = None,
+        dimensions: Dimensions = None,
+        period: Period = None,
+        unit: StandardUnit = None,
+    ) -> DescribeAlarmsForMetricOutput:
+        store = self.get_store(context.account_id, context.region)
+        alarms = [
+            a.alarm
+            for a in store.Alarms.values()
+            if isinstance(a, LocalStackMetricAlarm)
```

Comment:
q: can it be anything else than `LocalStackMetricAlarm`? I think not yet, is it for the future?

Reply:
yes, you are right, currently we only have `LocalStackMetricAlarm` in use. 
We prepared `LocalStackCompositeAlarm` as well in the model, put currently we don't use it. 
I would rather leave this here though, because we will likely forget about this, once we add the composite alarms 😅

Reply:
Of course yes! I wasn't asking to remove it 😄